src/request.ts

```diff
@@ -109,7 +109,8 @@ export class Request {
       const pattern = new RegExp(`{${key}\\??}`, 'g')
       const value = params[key]
       if (value != null && typeof value !== 'object') {
-        path = path.replace(pattern, encodeURIComponent(value))
+        const valueToReplace = this.methodDescriptor.disableParamsEncode ? value.toString() : encodeURIComponent(value)
```

Comment:
Let's see if we should change this to call `parameterEncoder` instead if it is provided, or whatnot, because I don't think it is wise to simply toString() it since receiving end might do a `decodeURI` on this and throw because of a `%` character.

Reply:
So, you're talking about this suggestion: https://github.com/tulios/mappersmith/pull/251#issuecomment-981429719

Reply:
Done https://github.com/tulios/mappersmith/pull/251/commits/c07250dcc55de810a29ad3e0a56872ea82049cf5